main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -358,7 +375,11 @@ private void loadAgencyData(GtfsTableContainer<GtfsAgency> agencyTable) {
     }
   }
 
-  private void loadFeedInfo(GtfsTableContainer<GtfsFeedInfo> feedTable) {
+  private void loadFeedInfo(
+      GtfsTableContainer<GtfsFeedInfo> feedTable,
+      GtfsTableContainer<GtfsTrip> tripContainer,
```

Comment:
With the change done earlier tripContainer, calendarTable and calendarDateTable are not used any more within this method, so they should be removed.

Reply:
removed!